sdcm/tester.py

```diff
@@ -123,6 +123,7 @@ def wrapper(*args, **kwargs):
         try:
             return method(*args, **kwargs)
         except Exception:
+            TEST_LOG.exception('Exception in wrapped method %s', method.__name__)
```

Comment:
Yep, my mistake, my brain read `error` instead of `exception`. Disregard the original comment.